packages/repository/test/unit/repositories/legacy-juggler-bridge.unit.ts

```diff
@@ -71,6 +71,13 @@ describe('DefaultCrudRepository', () => {
     await model.deleteAll();
   });
 
+  it('shares the backing PersistedModel across repo instances', () => {
+    const model1 = new DefaultCrudRepository(Note, ds).modelClass;
+    const model2 = new DefaultCrudRepository(Note, ds).modelClass;
+
+    expect(model1 === model2).to.be.true();
```

Comment:
Does deep assertion not work with `shouldjs`?

Reply:
We can use `expect(model1).to.be.exactly(model2);`

Reply:
We can. My experience is that equality assertion produces an error message that's rather unhelpful, because it it lists all model properties and properties of these properties. Because a model is attached to a datasource that has a reference to a model builder, eventually all models registered with the data source are printed in the diff. That's why I decided to use this form, because really all we want to see is whether the two models are the same V8 entity or not.